src/helpers.php

```diff
@@ -21,7 +22,7 @@ function current_route(string $locale = null): string
 
         return localized_route(
             Str::replaceFirst(locale().'.', null, $route->getName()),
-            $query,
+            array_merge($parameters, [$query]),
```

Comment:
This will not work because `[$query]` would cause a merge with `[null]` when `$query` is `null`, which can lead to unexpected results like this:

```php
$parameters = ['foo' => 'bar'];
$query = null;

array_merge($parameters, [$query]); // ['foo => 'bar', 0 => null]
```

So simply change to `array_merge($parameters, $query)`.